skia/RSkComponentRNSVGSvgView.cpp

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (C) 1994-2023 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+#include "experimental/svg/model/SkSVGRenderContext.h"
+
+#include "RSkComponentRNSVGGroup.h"
+#include "RSkComponentRNSVGSvgView.h"
+
+namespace facebook {
+namespace react {
+
+RSkComponentRNSVGSvgView::RSkComponentRNSVGSvgView(const ShadowView &shadowView)
+    : INHERITED(shadowView,LAYER_TYPE_PICTURE,SkSVGTag::kSvg) {}
+
+void RSkComponentRNSVGSvgView::OnPaint(SkCanvas *canvas) {
+    SkSVGLengthContext       lctx(svgContainerSize);
+    SkSVGPresentationContext pctx;
+    
+    printChildList();
```

Comment:
should this function call be under DEBUG mode

Reply:
Are you suggesting to have a Debug Mode define or to reuse any existing one( if exist).